xclim/indicators/land/_streamflow.py

```diff
@@ -12,44 +8,33 @@
 __all__ = ["base_flow_index", "freq_analysis", "stats", "fit", "doy_qmax", "doy_qmin"]
 
 
-class Streamflow(Indicator):
+class Streamflow(Daily):
     context = "hydro"
     units = "m^3 s-1"
     standard_name = "discharge"
 
     @staticmethod
-    def compute(*args, **kwds):
-        pass
-
-    def cfprobe(self, q):
-        cfchecks.check_valid(q, "standard_name", "streamflow")
+    def cfcheck(q):
+        check_valid(q, "standard_name", "water_volume_transport_in_river_channel")
 
 
 class Stats(Streamflow):
-    def missing(self, *args, **kwds):
-        """Return whether an output is considered missing or not."""
-        from functools import reduce
-
-        indexer = kwds["indexer"]
-        freq = kwds["freq"] or generic.default_freq(**indexer)
-
-        miss = (missing.missing_any(da, freq, **indexer) for da in args)
-        return reduce(np.logical_or, miss)
+    missing = "any"
 
 
 # Disable the missing value check because the output here is not a time series.
```

Comment:
Why this is not in docstring?

Reply:
Replaced "any" by "at_least_n" and set `missing_options`. Fixed bug related to hard-coded missing options.